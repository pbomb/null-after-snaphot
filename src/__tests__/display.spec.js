import React from 'react';
import { render } from 'react-testing-library';
import Display from '../display';

test('container is not null after snapshot', () => {
  const { container } = render(<Display text="test text" />);

  expect(container.firstChild).toMatchSnapshot();
  expect(container.firstChild).not.toBeNull();
});
