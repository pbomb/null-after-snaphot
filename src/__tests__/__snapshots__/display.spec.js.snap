// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container is not null after snapshot 1`] = `
.css-0,
[data-css-0] {
  color: #000;
  background-color: #fff;
}

<div
  class="css-0"
>
  test text
</div>
`;
